src/main/java/com/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.duke.command;
+
+import com.duke.util.Storage;
+import com.duke.task.Task;
+import com.duke.task.TaskList;
+import com.duke.util.Ui;
+
+public class DeleteCommand extends Command {
+
+    private int pos;
+
+    public DeleteCommand(int pos) {
+        this.pos = pos;
+    }
+
+    /**
+     * Execute the command to delete task in the list.
+     * @param tasks List of Task
+     * @param ui User interface
+     * @param storage Storage used by Duke
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        Task ts = tasks.remove(pos-1);
```

Comment:
Do consider adding space character around operators.
Can also consider renaming variable for readability?
```suggestion
        Task deletedTask = tasks.remove(pos - 1);
```